client/src/partials/patient_invoice/patientInvoice.html

```diff
@@ -100,6 +100,15 @@
       </div><!-- End (12) columns -->
     </div><!-- End row -->
 
+    <div ng-show="PatientInvoiceCtrl.patientBalance < 0" class="row">
+      <div class="col-xs-12">
+        <div class="alert alert-warning">
+          <strong class="text-capitalize">{{ 'CASH.CAUTION_REMAINING' | translate }}</strong>:
+          {{PatientInvoiceCtrl.patientBalance * -1 | currency}}
```

Comment:
I think this `currency` filter should take the enterprise currency ID as a parameter.